lib/coinbase/wallet.rb

```diff
@@ -84,54 +117,31 @@ def default_address
     # Returns the Address with the given ID.
     # @param address_id [String] The ID of the Address to retrieve
     # @return [Address] The Address
-    def get_address(address_id)
+    def address(address_id)
```

Comment:
This one feels a bit weird to me since we're already exposing `addresses`.

In what flow will someone have an on-chain address string that is associated with a wallet, where they already have the wallet in memory, and do not have a reference to the address object?

I suspect it might be more likely that they have an `address` without a wallet and want to be able to search for it and return it if the user/project owns the wallet and address?